charts/kueue/templates/manager/manager.yaml

```diff
@@ -25,9 +25,10 @@ spec:
         - --zap-log-level=2
         command:
         - /manager
+        {{- with .Values.controllerManager.manager.env  }}
```

Comment:
it doesn't look like we have this in `values.yaml`.
Is there a reason why users might need them?

Reply:
if we don't specify it in ```values.yaml```, it is equal to an empty array by default. I think it can make the chart flexible if we decide to add some variables in the future without changing the chart

Reply:
I'm more of a "don't add it until needed".
People might we wondering if they missed a configuration while we don't actually support any variables

Reply:
ok, let's remove it completely